examples/chrome-sandbox/test/e2e/benchmark_test.go

```diff
@@ -0,0 +1,274 @@
+// Copyright 2025 The Kubernetes Authors.
```

Comment:
Did you consider putting this test file under the top level `test/e2e` directory? That would probably help keep the e2e test entrypoint simple.

It would also probably be nice if this reused the existing e2e framework rather than reimplementing some of the boilerplate and test utilities

Reply:
I'll take a look.  This PR predates that framework, I believe